src/Kernel/CompiledCode.class.st

```diff
@@ -235,11 +235,11 @@ CompiledCode >> frameSize [
 { #category : #literals }
 CompiledCode >> hasLiteral: literal [
 	"Answer whether the receiver references the argument, literal."
-	2 to: self numLiterals - 1 do: "exclude superclass + selector/properties"
+	1 to: self numLiterals - self literalsToSkip do: "exclude superclass + selector/properties"
```

Comment:
Now it starts with 1. Do you know why it was 2 before?

Reply:
yes, is was 2 to - 1, now it is 1 to -2, but #literalAt: does "+1".